src/Microsoft.ApplicationInsights/DataContracts/RequestTelemetry.cs

```diff
@@ -255,10 +255,14 @@ void ITelemetry.Sanitize()
             this.Data.id = this.Data.id.SanitizeName();
             this.Data.id = Utils.PopulateRequiredStringValue(this.Data.id, "id", typeof(RequestTelemetry).FullName);
 
-            // Required field
+            // Required fields
             if (string.IsNullOrEmpty(this.ResponseCode))
             {
-                this.ResponseCode = "200";
+                this.ResponseCode = string.Empty;
+            }
+
+            if (!this.Success.HasValue)
```

Comment:
This changes our current experience in UI where requests may now have <blank> value if we were not able to collect response code. However, it's correct to respect that success is set to false. Can we set request to 200 when success if true and set to success to true only if it was not set to false before?